src/components/Suggestion/SuggestionListWrapper.vue

```diff
@@ -128,11 +153,26 @@ export default {
 
 	min-width: 200px;
 	max-width: 400px;
+	width: 80vw;
```

Comment:
Not sure if we really need it here. It seems to look ok without this line.

Reply:
This makes sure that on mobile devices there is still a bit of space around the picker